src/SuplaBundle/Entity/Schedule.php

```diff
@@ -115,6 +115,12 @@ class Schedule {
      */
     private $caption;
 
+    /**
+     * @ORM\Column(name="retry", type="boolean", nullable=false)
+     * @Groups({"basic"})
+     */
+    protected $retry = true;
```

Comment:
Don't you think `retryOnFailure` would be clearer?

Reply:
Refer to the last comment https://github.com/SUPLA/supla-cloud/issues/174#issuecomment-357701024